bundles/org.openhab.binding.ipcamera/src/main/java/org/openhab/binding/ipcamera/internal/DahuaHandler.java

```diff
@@ -169,18 +169,94 @@ private void processEvent(String content) {
                     ipCameraHandler.setChannelState(CHANNEL_TOO_BLURRY_ALARM, OnOffType.OFF);
                 }
                 break;
+            case "AccessControl":
+                if ("Pulse".equals(action)) {
+                    if (content.contains("\"Method\" : 1")) {
+                        startIndex = content.indexOf("CardNo", startIndex) + 11;
+                        if (startIndex > 0) {
+                            endIndex = content.indexOf(",", startIndex) - 1;
+                            String cardNo = content.substring(startIndex, endIndex);
+                            ipCameraHandler.setChannelState(CHANNEL_ACCEPTED_CARD_NUMBER, new StringType(cardNo));
+                            ipCameraHandler.setChannelState(CHANNEL_DOOR_UNLOCK, OnOffType.ON);
+                        }
+                    }
+                    if (content.contains("\"Method\" : 5")) {
```

Comment:
Should this be ELSE IF instead of IF? If it matches method1 do we need to keep checking?

Reply:
Oh yeah, fixed.

Reply:
It is not showing as changed in this PR, can you push the updated code or tell me if I am wrong, thanks.

Reply:
Not pushed, as I'm a bit stuck with other issue - what if we'll just organize additional channels to make channels easier to use for binding's user?
`		
<channel-group-type id="doorphoneProperties">
			<label>Doorphone Properties</label>
			<description>This is a group of channels relating to doorphone properties</description>
			<category>Door</category>
			<channels>
				<channel id="acceptedCardNumber" typeId="acceptedCardNumber"/>
				<channel id="doorUnlock" typeId="doorUnlock"/>
				<channel id="doorContact" typeId="doorContact"/>
				<channel id="exitButton" typeId="exitButton"/>
				<channel id="motionDetectionLevel" typeId="motionDetectionLevel"/>
			</channels>
		</channel-group-type>
`
This way I will just implement all VTO's additional channels. There will be a plenty of them, but they will be at the bottom and categorized.